Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -58,14 +58,19 @@ def get_fetch_times(last_fetch):
 
 class Client:
     def __init__(self, proofpoint_url, api_version, verify, service_principal, secret, proxies):
-        self.base_url = "{}/{}/siem".format(proofpoint_url, api_version)
+        self.base_url = urljoin(proofpoint_url, api_version)
```

Comment:
no need for extra `base_url_no_api_version `
just keep the base as it is and save the `self.api_verison = api_version`.
in the http_request pass `forensics_api` of type boolean
```python
if forensics_api:
    full_url = urljoin(self.base_url, '/v2/forensics')
else:
    full_url = urljoin(urljoin(self.base_url, self.api_version), url_suffix)
```